src/test/java/uk/gov/companieshouse/officerfiling/api/validation/OfficerUpdateValidatorTest.java

```diff
@@ -473,15 +473,15 @@ void testDoesNameMatchChipsDataWhenItDoesNotMatchesChipsData() {
     @Test
     void testValidateNameSectionWhenMatchedChipsDataShouldReturnValidationError() {
         // Given
-        when(companyAppointment.getTitle()).thenReturn("Mr");
+        when(companyAppointment.getTitle()).thenReturn("Mr.");
         when(companyAppointment.getForename()).thenReturn("John");
         when(companyAppointment.getOtherForenames()).thenReturn("Doe");
         when(companyAppointment.getSurname()).thenReturn("Smith");
 
         final var officerFilingDto = OfficerFilingDto.builder()
                 .referenceEtag(ETAG)
                 .referenceAppointmentId(FILING_ID)
-                .title("Mr")
```

Comment:
Maybe keep this unit test for "Mr" and add another for "Dr." as that is the one special case we added?

Reply:
Another test added